src/farajaland/features/languages/generated/client/client.json

```diff
@@ -269,6 +269,7 @@
         "constants.nationalSystemAdmin": "System admin (national)",
         "constants.noNameProvided": "No name provided",
         "constants.noResults": "No result to display",
+        "constants.noResult": "No result",
```

Comment:
Do we need another message for this. Or we can change the existing one. @jpye-finch do you have any thought?

Reply:
I think we can merge noResults and noResult.   just have "constants.noResults": "No results"